lib/vmdb/loggers/multicast_logger.rb

```diff
@@ -0,0 +1,32 @@
+class MulticastLogger < Logger
+  attr_accessor :loggers
+
+  def initialize(*loggers)
+    require 'set'
+    @loggers = Set.new
```

Comment:
My concern is that you wouldn't want the same logger added more than once to avoid getting duplicate messages.